azure-cli-samples/create-run-test-yaml/README.md

```diff
@@ -0,0 +1,73 @@
+---
+page_type: sample
+languages:
+- bash
+- azurecli
+products:
+- azure
+- azure-load-testing
+urlFragment: azure-load-testing-create-with-yaml-azure-cli
+description: Create and run load test with load test with load testing YAML config file from Azure CLI extension for Azure Load Testing
+---
+
+# Create and run load test with load test with load testing YAML config file by using the Azure CLI extension for Azure Load Testing
+
+This project demonstrates how to use an Azure CLI sample script to create and run a load test with [load testing YAML config file](https://learn.microsoft.com/azure/load-testing/reference-test-config-yaml) on Azure Load Testing.
+
+In this sample script we perform the following operations:
+
+1. Create a Load Testing resource
+1. Create a load test with load testing YAML config file
+1. Add app components to the test for monitoring
+1. Add server metrics for the above app component
+1. Run the test by creating a test run
+1. Download result files for the test run
+
+## Prerequisites
+
+1. An existing Azure subscription. If you don't already have a Microsoft Azure subscription, you can create one for free [here](http://go.microsoft.com/fwlink/?LinkId=330212).
+1. An existing resource group.
+1. Azure CLI. You can install by following the steps mentioned [here](https://docs.microsoft.com/cli/azure/install-azure-cli).
+
+## Getting started
+
+### Preparation
+
+1. Open the source code in VS Code.
+
+1. Update the `create-run-test-yaml.sh` file and replace the text placeholders:
+
+    | Placeholder | Description |
+    | ----------- | ----------- |
+    | `<subscriptionId>` | Azure subscription Id |
+    | `<myRG>` | Resource group that contains the Azure Load Testing resource. |
+    | `<myLoadTestResource>` | Name of the Azure Load Testing resource.  |
+    | `<myLoadTest>` | Unique name for the load test, must contain only lower-case alphabetic, numeric, underscore or hyphen characters. |
+
+1. Update the path for the YAML file in `loadTestConfig`.
+
+1. Update the app component(s) and server metric(s) for the target application you are testing.
```

Comment:
How can people find these values? Do they need to use the Azure Resource Explorer or something else?